### DISCUSSION TITLE
Changed password and cant get in now because i dont have recovery key. HELP

### DISCUSSION THREAD

**Discussion by Roy001-sudo** [ente-io/ente#3501] [Q&A]
Hi, 
I decided to change my password on Ente photos because i thought the password would be different from Ente Auth. However when i changed the password, it also changed Ente auth. Now i am locked out from Ente photos and Ente Auth. It requires a Auth app or the recovery key in order for me to get access. I thought i had saved the recovery key and it seems like i am out of luck. I cant use a auth app because i have it connected to the Ente auth account.  Does anyone know or suggest what i should do?

**Comment by Roy001-sudo** [comment]
Could yall temporarily allow it so that a verification email can be sent to the email so that one can get access again even if it is not enabled. But can type in the email to the account that it is associated to

**Comment by Damariobros** [comment]
Do you mean to say that you have two-factor enabled on your account, and that two-factor token is saved in Ente Auth?

If that's the case, and you still know the password that you changed it to, send an email to [support@ente.io](mailto:support@ente.io) from the email you used to setup your account, and explain the situation. They can disable two-factor on your account for you, and you can then use your password to login.

Next time, do not store your Ente account's two-factor token inside Ente Auth. This creates a chicken-and-the-egg situation, which can lead to a lockout. Store it in a different authenticator application, or use another method such as Email Verification (you can enable this from the mobile and desktop apps) or Passkeys (store these in a separate password manager).

In addition, be sure to save your Recovery Key to a text file, or write it down, and store it in a secure location. Multiple copies in different locations would be preferable (e.g. one in a different cloud provider not accessed with Ente Auth, one on a flash drive, one on paper in a safe). Do not store it on your computer in an unencrypted file.


Your Ente account is used to access all of Ente's services, much like how your Google account is used to access all of Google's services, or Proton accounts are used for all Proton services. It uses the same password. If you wish to secure your Ente Photo and Ente Auth apps separately and with unique passwords, you may use the App Lock feature included in the desktop and mobile apps. Your account will remain signed in, but the app will be locked and encrypted until you enter the PIN or Password you set for that app lock.

**Reply by Roy001-sudo** [reply]
Thank you very much. I did as you described. I followed your suggestion and got the help. Thank you very much.
